docs/source/dataset.rst

```diff
@@ -0,0 +1,42 @@
+===============
+Sample Dataset
+===============
+Public datasets
+~~~~~~~~~~~~~~~~
+Some data to work with have been obtained from _`Zenodo <https://zenodo.org>`. 
```

Comment:
Hey @sayandip18 , for attaching the external URIs consider following example :
#### Plain Text
" External hyperlinks, like \`Python \<http://www.python.org/ \>\`_  "
#### Result
"External hyperlinks, like [Python](http://www.python.org)  "

Reply:
Thanks! Fixed it.